Source/WebCore/platform/network/cocoa/WebCoreNSURLSession.h

```diff
@@ -132,7 +132,7 @@ WEBCORE_EXPORT @interface WebCoreNSURLSession : NSObject {
 @interface WebCoreNSURLSessionDataTask : NSObject {
     WeakObjCPtr<WebCoreNSURLSession> _session; // Accesssed from operation queue, main and loader thread. Must be accessed through Obj-C property.
     RefPtr<WTF::WorkQueue> _targetQueue;
-    RefPtr<WebCore::PlatformMediaResource> _resource; // Accesssed from main and loader thread. Must be accessed through Obj-C property.
+    RefPtr<WebCore::PlatformMediaResource> _resource; // Accesssed from loader thread.
```

Comment:
Can we add a lock annotation?

Reply:
they have no effect with objC unfortunately.

Reply:
so I placed an assert on the setter and accessor instead